general/clients/skins.md

```diff
@@ -5,7 +5,7 @@ title: CSS Customization
 
 # CSS Customization
 
-[Custom CSS](https://www.w3schools.com/cssref/css3_browsersupport.asp) provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. Currently, the CSS modifications only work on the web client. To implement these changes, go to Dashboard > General > Custom CSS.
+[Custom CSS](https://www.w3schools.com/cssref/css3_browsersupport.asp) provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. Currently, the CSS modifications only work on the web client. The code will apply the code in order that it is written so code can override previously stated custom CSS. To learn more see [CSS Specificity](https://www.w3schools.com/css/css_specificity.asp). To implement these changes, go to Dashboard > General > Custom CSS.
```

Comment:
We should link to MDN instead of w3schools. https://developer.mozilla.org/en-US/docs/Web/CSS

Reply:
Done, changed the main link. Left the Specificity link as is since it's a good read.